types/stripe/index.d.ts

```diff
@@ -3472,6 +3473,8 @@ declare namespace Stripe {
              * Stripe will attempt to pay this invoice using the default source attached to the
              * customer. When sending an invoice, Stripe will email this invoice to the customer
              * with payment instructions.
+             *
+             * This field has been renamed to collection_method and will be removed in a future API version.
```

Comment:
An `@deprecated` tag here would be a good idea.

Reply:
Fixed